.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
+  build_and_test:
+    name: Build and test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [10, 12]
```

Comment:
Do we want that? I see that it was in Travis configuration but I don't see why we would like to test it with old version of `node`

Reply:
I thought that maybe there is some reasoning behind this. If there is no need to test the old version then I will remove it.
@TomWoodward

Reply:
because this is a library i think its helpful to do the multiple node versions, as long as the tests don't take too long i think its good